<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanCard</class>
 <widget class="QWidget" name="PlanCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;background:#252526; border-radius:10px;color:white;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>240</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>240</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="tipEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>132</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#3a7cff</string>
    </property>
    <property name="text">
     <string>2016/10/12~2016/10/21</string>
    </property>
   </widget>
   <widget class="FCircle" name="typeCircle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="color">
     <string>#d90d0d</string>
    </property>
    <property name="flag">
     <string>紧急事件</string>
    </property>
    <property name="radius">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="titleLab">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>32</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>This is a Title</string>
    </property>
   </widget>
   <widget class="QFrame" name="toolFrame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>79</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ccc;border-radius:0px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="delBtn">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>0</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#delBtn:hover{background:#666;}</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="completeBtn">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#completeBtn:hover{background:#666;}</string>
     </property>
     <property name="text">
      <string>标记为完成</string>
     </property>
    </widget>
   </widget>
   <zorder>typeCircle</zorder>
   <zorder>titleLab</zorder>
   <zorder>tipEdit</zorder>
   <zorder>timeLab</zorder>
   <zorder>toolFrame</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FCircle</class>
   <extends>QLabel</extends>
   <header>fcircle.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
